cmd/oneshot.go

```diff
@@ -13,8 +13,8 @@ import (
 	"github.com/google/subcommands"
 )
 
-// EvalCmd struct with flags.
-type EvalCmd struct {
+// OneshotCmd struct with flags.
```

Comment:
Oneshot might be misleading if/when we add support to stop and start migrations.
I think being descriptive and saying schema-and-data in this case might make more sense.

Reply:
Yes, I've also been thinking about this a bunch too and have downloaded PR218 and have been playing with the command-line interface.

I apologize Asheesh -- when we discussed this previously, I said oneshot was ok, but I'm less sure now.

The problem with oneshot is that it isn't clear what it does. Moreover, oneshot has some misleading connotations e.g. the standard definition of one-shot is "something that happens only once" (e.g. https://dictionary.cambridge.org/dictionary/english/one-shot). We're using oneshot to mean "all-in-one" but that's a less common usage I think.

I know I'm backtracking a bit here, but I agree with Aditya. I think schema-and-data is better (even if its too long).
Some alternative options:
 schema-and-data
 schema+data
 schema-data
My quick take on these:
 schema-and-data: a solid option, but kind of verbose
  schema+data: a good option, but is it ok to have "+" in the command name?
  schema-data: an ok option, but potentially confusing: do you mean the schema's data?

Reply:
I didn't have a preference between oneshot and schema-and-data, but agree that schema-and-data is clearer. Have updated the subcommand and it's references in the READMEs to say schema-and-data now.